app/Http/Controllers/SpecializationController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\SpecializationResource;
+use App\Services\SpecializationService;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class SpecializationController extends Controller
+{
+    protected $specializationService;
+
+    public function __construct(SpecializationService $specializationService)
+    {
+        $this->specializationService = $specializationService;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $specializations = $this->specializationService->index();
+        return response()->json(SpecializationResource::collection($specializations), Response::HTTP_OK);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
```

Comment:
Delete all unused functions in all classes in the project

Reply:
done